web/client/components/import/style/StylePanel.jsx

```diff
@@ -0,0 +1,198 @@
+const PropTypes = require('prop-types');
```

Comment:
Is this completely different than the styling panel used by the Shapefile plugin?

Reply:
No, it is the same, but more isolated from import stuff, so we may be able to improve and reuse it in a easier way.